### PR TITLE
Add ArcGISOnline provider

### PR DESCRIPTION
**Pull request by robertd**
I'll revisit tests once #25 has been addressed.
Example works once you put in your client id & secret.

Cheers

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ var internals = {};
 
 
 exports.register = function (server, options, next) {
-
```
Comment (geek) [inline lib/index.js]: we use newlines after each new scope

Comment (robertd) [inline lib/index.js]: I will revert that one

**Comment by robertd** [comment]
Squashed and force-pushed

**Comment by geek** [comment]
This looks good, please add this to the list in the README: https://github.com/hapijs/bell#options

**Comment by geek** [comment]
@robertd can't be merged, please update

**Pull request closed without merging**